public/src/components/edit_account.jsx

```diff
@@ -72,7 +73,16 @@ class EditAccountInfo extends Component {
           label="Phone Number"
           component={this.renderField}
         />
-        <button type="submit">Update</button>
+        <div className="row">
```

Comment:
I think there might be bootstrap-react button components that we could use. 
https://react-bootstrap.github.io/components.html